### DISCUSSION TITLE
Destruction Warlock Rain of Fire

### DISCUSSION THREAD

**Discussion by Rafa28062013** [Hekili/hekili#927] [Q&A] [answered]
Hello:)

Why doesn't the addon show me when to use Rain of Fire?

**Comment by Hekili** [comment, marked as answer]
It does.  You'll need to familiarize yourself with how the addon does its target detection for ranged specs.  Enemies don't count until you've hit them with something.  The Destruction APL uses Rain of Fire at 3+ targets (plus some other conditionals).

You can also use a dual display mode that has the AOE display active, to see multi-target recommendations and single target recommendations at the same time.

**Reply by Rafa28062013** [reply]
Hello, thanks for youre answer. But i have Dual Mode, but they didnt tell me when i have to use Rain of Fire. I dont know why not.

**Reply by Hekili** [reply]
I don't either.  You'd need to submit an issue ticket with all the required information (including snapshots) to know why you received a specific set of recommendations.
